api/exec/kubernetes_deploy.go

```diff
@@ -251,3 +162,21 @@ func (deployer *KubernetesDeployer) ConvertCompose(data []byte) ([]byte, error)
 
 	return output, nil
 }
+
+func (deployer *KubernetesDeployer) getEdgeUrl(endpoint *portainer.Endpoint) (string, error) {
```

Comment:
Why EE branch just uses `getAgentURL` instead of `getEdgeUrl` for both agent and edge endpoints?
If possible try to keep consistency between EE and CE.

Reply:
make it constant to EE